src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -222,6 +223,25 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.NuGetPackageSourceUrl", ConfigurationEntryScope.ServerWideOnly)]
         public string NuGetPackageSourceUrl { get; set; }
 
+        [Description("Determine how RavenDB should coordinate expensive journal writes for indexes. This is used to reduce I/O contention with many indexes.")]
+        [DefaultValue(IndexWriteSynchronizationMode.Database)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.JournalSynchronizationMode", ConfigurationEntryScope.ServerWideOnly)]
+        public IndexWriteSynchronizationMode JournalSynchronizationMode { get; set; }
+
+        [Description("Maximum allowed concurrent writes for the indexes")]
+        [DefaultValue(4)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.JournalMaxConcurrentWrites", ConfigurationEntryScope.ServerWideOnly)]
+        public int JournalMaxConcurrentWrites { get; set; }
+
+        [Description("Maximum size in Megabytes of the concurrent writes for the indexes")]
+        [DefaultValue(128)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.JournalMaxConcurrentWritesSizeInMb", ConfigurationEntryScope.ServerWideOnly)]
+        public int JournalMaxConcurrentWritesSizeInMegabytes { get; set; }        
```

Comment:
Use `Size` type e.g.
https://github.com/ravendb/ravendb/blob/785eb6a41c1faf45331cfe17ebe8a216d0845c6a/src/Raven.Server/Config/Categories/StorageConfiguration.cs#L54-L58

Reply:
The property then should be `JournalMaxConcurrentWritesSize` and the configuration entry `Indexing.JournalMaxConcurrentWritesSizeInMb`